templates/blog/index.html

```diff
@@ -0,0 +1,29 @@
+{% extends "base.html" %}
+{% block content %}
+  <ul>
+    {% for post in post_list %}
+      <li><a href="{% url 'blog:post-detail' pk=post.pk %}">{{ post.title }}</a></li>
+      <p>{{ post.created_time }}</p>
+      <p>{{ post.owner }}</p>
+      <p>{{ post.content }}</p>
+      <p>commentaries: {{ post.commentaries_count }}</p>

```

Comment:
There is a typo in 'commetaries'. It should be 'commentaries' to match the annotated field name in the view.